conf/nginx.conf

```diff
@@ -30,4 +30,6 @@ location YNH_WWW_PATH {
 
   # Include SSOWAT user panel.
   include conf.d/yunohost_panel.conf.inc;
-}
+  
+# Very important security restriction
+location /Cheky/var { deny all;}
```

Comment:
I think you should only specify `/var` and put the missing `}` afterward because:
- `/Cheky` isn't necessarily what users will have chosen
- there's a syntax error because of that missing bracket